<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生成绩管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>排序方式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="value">
         <item>
          <property name="text">
           <string>学号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>成绩</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond">
         <item>
          <property name="text">
           <string>升序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>降序</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="StudentTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>学生成绩：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>学生姓名：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="scoreEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="IDEdit"/>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>学生学号：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="insertBtn">
         <property name="text">
          <string>插入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBtn">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyBtn">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StudentTableView</class>
   <extends>QTableView</extends>
   <header>D:/cppDesign/EasyChat/studenttableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
